### PR TITLE
Updated and added comments

### PR DESCRIPTION
**Pull request by Abdiwali1**

### CODE REVIEW

**Review by untuzina — APPROVED** [review]

File: src/Practice01.java [hunk 1]
```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
 
         System.out.println("Hello Cydeo friends!");
         System.out.println("Good luck to everyone!");
+        System.out.println("Good trials");
```
Comment (stars2026) [inline src/Practice01.java:9]: Thank you for the comment

File: src/Practice01.java [hunk 2]
```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
...
 
     }
 }
```
Comment (stars2026) [inline src/Practice01.java:11]: Reviewed

**Review by stars2026 — APPROVED** [review]

File: src/Practice01.java (hunk 2 above)
Comment (Abdiwali1) [reply src/Practice01.java:11]: Reveiwed

**Review by stars2026 — COMMENTED** [review]
Good contribution to the project

**Review by Abdiwali1 — COMMENTED** [review]
All is well,

**Pull request merged**